### ISSUE TITLE
Stabilize performance of exact excerpt counting

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Queue#1273]
Caused by #1203 
It seems this check works in disabling way currently:
```
        if (((Byteable) index2indexArr).bytesStore() != null)
            return 0;
```
Leading to having 0 returned every time, and full scan every time.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.24ea7](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.24ea7), [BOM-2.24ea33](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.24ea33)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.23.40](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.23.40), [BOM-2.23.151](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23.151)
